### DISCUSSION TITLE
Katex 无法适应小屏幕问题解决方案

### DISCUSSION THREAD

**Discussion by luweizheng** [wangchucheng/hugo-eureka#70] [Q&A] [answered]
你好，我在Latex的[Issue](https://github.com/KaTeX/KaTeX/issues/1023)里看到了你修复了公式太长的问题。比如下图，在iPhone上，因为公式太长，宽度超过了文本框的宽度。我不怎么了解CSS，所以没怎么看懂最近的那个fix，而且我用的是Jekyll，可能和Hugo稍有不同。所以请教一下，Katex如果公式太长，目前的解决方案是啥呢？CSS里需要设置哪几个参数呢？

![image](https://user-images.githubusercontent.com/57242864/108601063-95008680-73d5-11eb-830f-8aedf8d3f6ca.png)

**Comment by wangchucheng** [comment]
请问你所发的图片是Eureka中目前仍存在的问题吗？

**Reply by luweizheng** [reply]
> 请问你所发的图片是Eureka中目前仍存在的问题吗？

不是Eureka的问题，是我用的一个别的theme，出现了截图中的这种情况，所以想咨询咨询。如果不方便回答也不要紧。

**Comment by wangchucheng** [comment, marked as answer]
我在相应的commit中只添加了一个关于overflow的属性，使用该属性即可隐藏溢出部分。可参考该commit的代码查看具体css属性，祝好！


________________________________
发件人: luweizheng <notifications@github.com>
发送时间: Sunday, February 28, 2021 12:08:28 PM
收件人: wangchucheng/hugo-eureka <hugo-eureka@noreply.github.com>
抄送: C. Wang <wangchucheng@outlook.com>; Comment <comment@noreply.github.com>
主题: Re: [wangchucheng/hugo-eureka] Katex 无法适应小屏幕问题解决方案 (#70)


请问你所发的图片是Eureka中目前仍存在的问题吗？

不是Eureka的问题，是我用的一个别的theme，出现了截图中的这种情况，所以想咨询咨询。如果不方便回答也不要紧。

―
